pipeline/pipeline-jobs.yml

```diff
@@ -0,0 +1,46 @@
+resources:
+- name: census-worth-self-help
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/census-worth-self-help.git
+    branch: feature/ONS-152-pipeline-cleanup
+  webhook_token: ((git.census-worth-self-help-webhook-token))
+  check_every: 15m
+
+- name: content-repo
```

Comment:
"content-repository" not "content-repo" please

Reply:
done